### ISSUE TITLE
Investigate: Harrison St and Folsom St, SF

### ISSUE THREAD

**Issue by migurski** [pelias/interpolation#18]
Addresses seem clustered in the north east, none exist down in the Mission District.

**Comment by missinglink** [comment]
this is due to limiting the results to 500 [here](https://github.com/pelias/interpolation/blob/master/query/address.js#L6).

I tried changing it to 1500 and it resulted in the image below:

![folsom](https://cloud.githubusercontent.com/assets/738069/19685073/e9248106-9aba-11e6-956b-d6d47e37d183.png)

I'm going to leave this open, it's important for roads like this that:

- all the *relevant* point data is used when considering interpolation (this may only need to be 2 points)
- performance considerations are addressed for roads with >1000 points.

There is a secondary issue which is how many points get returned when using the demo; which I don't think is a bit deal, because it's not meant to be used in a production setting.

**Comment by missinglink** [comment]
What we really need here is a SQL query that says:

> "I want a single row above and a single row below housenumber X"

This will give the ideal performance, so if we have say.. 

```
housenumber 10
housenumber 11
housenumber 14
housenumber 20
housenumber 12
```

and we want to interpolate `12`, then the query should only return two rows:

```
housenumber 11
housenumber 14
```

@migurski do you know a simple way to do this is SQL?

**Comment by missinglink** [comment]
related code comment: https://github.com/pelias/interpolation/blob/master/api/search.js#L42

**Comment by missinglink** [comment]
fixed in #22
